### ISSUE TITLE
New Work : Model Implementation (Model -II)

### ISSUE THREAD

**Issue by eeshan15** [The-Red-Wood-Lab/Micro-Gas-Turbine-electrical-energy-Prediction#8]
## Model Description 
**Multivariate Regression** : Multivariate Regression is a type of Linear Regression in which there are multiple dependent columns and only one independent column so that column will become the target variable. 

So Far we have applied the XGBOOST regression algorithm and now this is our second model. We will be trying different models and choose the best one.

## PR Submission details
1) Download the dataset (if you haven't). You can find the links in the issue number 4 :  #4 
2) Apply the Multivariate Regression model. Do not worry if you don't know how to apply here is an article for you from where you can learn the multivariate regression. click over this link : [Article](https://www.geeksforgeeks.org/multivariate-regression/)
3) Calculate the RMSE and R2 Score. Here are the articles for that as well If you want to learn it. The cost Function is as same as that of Linear regression. This article tells best about it. Click over this link : [Article](https://www.geeksforgeeks.org/ml-linear-regression/#cost-function-for-linear-regression)
4) For Successful submission of the PR you have to the Screen shots of the graphs which tells the relation predicted values of the el_power column based on other dependent columns which is time and voltage. Below is the description :
      1. Take the training dataset and take 75:25 ratio convert 25% of training into Validation dataset. You can also use K-Fold Cross validation method.
      2. Next Check for overfitting by Visualizing training and Validation loss by plotting them in one single graph Graph. If your Validation Curve is going up and Training curve going down then there is a problem you have to check for the issue. You can use Regularization techniques etc to solve the problem.
5) Please show the Following scores in your PR. Needed Scores are shown in the Checklist section. 
 
## Checklist : 
- [ ] Applied The Multivariate Regression
- [ ] Applied Regularization to Fix overfitting if exist
- [ ] Applied screenshots of the Graph
- [ ] Showcased your Train , Validation and Test RMSE and R2 Scores 

## Conditions
A group of people will be working on the issue so the best model will be evaluated on the basis of Lowest RMSE Score Amongst them and R2 Score close to 1 its kind of leaderboard system. If your model is showcasing some overfitting then the PR will be rejected.

**Comment by yuvam2005** [comment]
Please assign me this task

**Comment by eeshan15** [comment]
Assigned to @yuvam2005 kindly proceed

**Comment by Hotchapu13** [comment]
Interested. Please assign to me

**Comment by Divyansh0108** [comment]
Assign please.

**Comment by eeshan15** [comment]
Assigned to both @Hotchapu13 and @Divyansh0108. Please Proceed as per instructed

**Comment by shriyans2912** [comment]
Interested. Please assign this to me.

**Comment by eeshan15** [comment]
Issue assigned to @shriyans2912
